std/protocols/permutation.asm

```diff
@@ -0,0 +1,42 @@
+use std::prover::challenge;
```

Comment:
I think at some point we should have `pub` as well, so we don't clutter the exported module contents here.